### DISCUSSION TITLE
How to deal with an old unmaintained port that needs a patch?

### DISCUSSION THREAD

**Discussion by diehard2** [microsoft/vcpkg#30261] [General]
I'm stuck using 1.5.1 of opentracing, a project which was deprecated a few years ago. Its a dependency of another project I have, and that one can't work with 1.6. That version is failing with versions of GLIBC greater than 1.34 for something that's a trivial patch, but I can't think of the best way to do this. At the moment, I've created a file based registry. I tried to give it my own version with the trivial patch, but it appears vcpkg prefers the official versions over mine. So now, I've settled for naming it opentracing-mycompany.

Is there an easier way to get an older version working? I think the ideal workflow is I could specify a version and port in my registry and if they don't conflict it would choose mine

**Comment by autoantwort** [comment]
You could create an [overlay port](https://learn.microsoft.com/en-us/vcpkg/users/registries#overlays), that is probably easier than creating a registry
